nodejs/week2/meal_app/src/backend/routes/meals.js

```diff
@@ -0,0 +1,90 @@
+const express = require("express");
+const app = express();
+
+const mealsJson = require(__dirname + '/../data/meals.json');
+const reviewsJson = require(__dirname + '/../data/reviews.json');
+
+mealsJson.forEach(meal => {
+    meal.review = reviewsJson.filter(mealReview => mealReview.mealId === meal.id)
+});
+
+app.get("/", (req, res) => {
```

Comment:
Please check my comment here:

https://github.com/Mekdes1/hyf-homework/pull/22#discussion_r387193511

Reply:
@hatko I am not able to crack this. Could you please tell how to do it?